apps/charterafrica/src/payload/blocks/Hero.js

```diff
@@ -0,0 +1,209 @@
+
+const blendModeOptions = [
+  {
+    label: "Normal",
+    value: "normal",
+  },
+  {
+    label: "Multiply",
+    value: "multiply",
+  },
+  {
+    label: "Screen",
+    value: "screen",
+  },
+  {
+    label: "Overlay",
+    value: "overlay",
+  },
+  {
+    label: "Darken",
+    value: "darken",
+  },
+  {
+    label: "Lighten",
+    value: "lighten",
+  },
+  {
+    label: "Color Dodge",
+    value: "color-dodge",
+  },
+  {
+    label: "Color Burn",
+    value: "color-burn",
+  },
+  {
+    label: "Hard Light",
+    value: "hard-light",
+  },
+  {
+    label: "Soft Light",
+    value: "soft-light",
+  },
+  {
+    label: "Difference",
+    value: "difference",
+  },
+  {
+    label: "Exclusion",
+    value: "exclusion",
+  },
+  {
+    label: "Hue",
+    value: "hue",
+  },
+  {
+    label: "Saturation",
+    value: "saturation",
+  },
+  {
+    label: "Color",
+    value: "color",
+  },
+  {
+    label: "Luminosity",
+    value: "luminosity",
+  },
+  {
+    label: "Initial",
+    value: "initial",
+  },
+  {
+    label: "Inherit",
+    value: "inherit",
+  },
+  {
+    label: "Unset",
+    value: "unset",
+  },
+  {
+    label: "Revert",
+    value: "revert",
+  },
+  {
+    label: "Revert Layer",
+    value: "revert-layer",
+  },
+];
+
+const Hero = {
+  slug: "hero",
+  fields: [
+    {
+      name: "slides",
+      type: "array",
+      minRows: 1,
+      fields: [
+        {
+          name: "title",
+          type: "group",
+          required: true,
+          fields: [
+            {
+              name: "color",
+              type: "text",
+              required: true,
+              localized: false,
+            },
+            richText({
+              name: "content",
+              required: true,
+              admin: {
+                elements: ["h2", "h3", "h4", "h5", "h6", "link"],
+                leaves: ["bold", "italic", "underline", "code"],
+              },
+              localized: true,
+            }),
+          ],
+        },
+        {
+          name: "subheading",
+          type: "group",
+          required: true,
+          fields: [
+            {
+              name: "color",
+              type: "text",
+              required: true,
+              localized: false,
+            },
+            {
+              name: "content",
+              type: "text",
+              required: true,
+              localized: true,
+            },
+          ],
+        },
+        {
+          name: "background",
+          type: "group",
+          required: true,
+          fields: [
+            {
+              name: "src",
+              label: {
+                en: "Background Image",
+                fr: "Image de fond",
+                pt: "Imagem de fundo",
+              },
+              type: "upload",
+              relationTo: "media",
+              required: true,
+            },
+            {
+              name: "color",
+              type: "text",
```

Comment:
@kilemensi What is the best option to ensure correct values for color? Do we use some validation function or swap it out for a custom [color picker component](https://payloadcms.com/docs/admin/components)

Reply:
That custom component doesn't look very useful to me i.e. it doesn't have color wheel or anything to use to pick colors @kelvinkipruto. A user still needs to type it.

For now I think the only thing we can add is a [validation](https://payloadcms.com/docs/fields/overview#validation) method that checks ([regex](https://stackoverflow.com/a/1636354)) the value is indeed hex **ARGB**